src/sql/src/plan/error.rs

```diff
@@ -385,6 +389,7 @@ impl fmt::Display for PlanError {
             },
             Self::InvalidPrivatelinkAvailabilityZone { name, ..} => write!(f, "invalid AWS PrivateLink availability zone {}", name.quoted()),
             Self::InvalidSchemaName => write!(f, "no schema has been selected to create in"),
+            Self::ItemAlreadyExists { name, item_type } => write!(f, "catalog item '{name}' : {item_type}, already exists"),
```

Comment:
Nit: instead of `'{name}'` I think it's better to do `write!(f, "catalog item '{}' : {item_type}, already exists", name.quoted())`.
`str.quoted()` handles escaping special characters nicely, which is particularly relevant for object names, since they are user inputted. It also applies double quotes, which our style guide recommends.

For reference, here's the SQL error style guide https://github.com/MaterializeInc/materialize/blob/main/doc/developer/style.md#error-message-style

Reply:
Ah ha, nice! I'll update the error message